src/test/java/com/team4/testingsystem/controllers/TestsControllerIntegrationTest.java

```diff
@@ -73,28 +72,26 @@ void destroy() {
     @Test
     void getUsersTestsSuccess() throws Exception {
         com.team4.testingsystem.entities.Test test1 = EntityCreatorUtil.createTest(user);
+        test1.setCreatedAt(null);
         testsRepository.save(test1);
 
         com.team4.testingsystem.entities.Test test2 = EntityCreatorUtil.createTest(user);
+        test2.setCreatedAt(null);
```

Comment:
Let's check all the values are passed

Reply:
There's a problem with comparing `LocalDateTime` after saving it to DB, so they are not equal...
This can be solved later, in PR where `createdAt` should be not null
I think we can switch to DTOs now, so that frontend can parse our responses, and fix this later :)